grails-app/services/au/org/ala/bie/IndexService.groovy

```diff
@@ -30,18 +29,21 @@ class IndexService implements GrailsConfigurationAware {
     SolrQuery suggestTemplate
     List<String> searchFq
     String searchDefType
+    String searchBoost
     List<String> searchQf
 
     @Override
     void setConfiguration(Config config) {
         def search = config.solr.search
+        searchBoost = search.boost
+        searchDefType = search.defType
         searchTemplate = new SolrQuery()
         search.fq.each { searchTemplate.addFilterQuery(it) }
-        searchTemplate.set('defType', search.defType)
+        searchTemplate.set('defType', searchDefType)
         searchTemplate.set('qf', search.qf.join(' '))
         search.bq.each { searchTemplate.add('bq', it) }
         searchTemplate.set('q.alt', search.qAlt)
-        searchTemplate.set('boost', search.boost)
+        searchTemplate.set('boost', searchBoost)
```

Comment:
Same question of checking null

Reply:
Same as above. See https://github.com/AtlasOfLivingAustralia/bie-index/blob/master/grails-app/conf/application.yml#L235